examples/annotation_import/basics.ipynb

```diff
@@ -78,12 +78,22 @@
     },
```

Comment:
<p>Should we delete this warning?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/883/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>